all-imports.html

```diff
@@ -1,3 +1,3 @@
 <!-- all-imports.html -->
 <link rel="import" href="i18n-msg.html">
-<link rel="import" href="i18n-page-example.html">
+<link rel="import" href="demo/i18n-msg-behavior-example.html">
```

Comment:
@RoXuS I've added another pull request for you that should correct the documentation. I don't think we need to include the demo as part of the all-imports.html file since that is just an example file.

I had put the docs for the behavior in the wrong place to have them included so I fixed that and it should show the correct docs.